README.md

```diff
@@ -392,43 +388,41 @@ relation. There are also peer relations and subordinate relations. While in the
 the data access rules and the consistency constraints on them are very different. For example, it does not make sense
 for a peer relation to have a different 'remote app' than its 'local app', because it's the same application.
 
-### PeerRelation
+### Peer Relations
 
-To declare a peer relation, you should use `scenario.state.PeerRelation`. The core difference with regular relations is
+To declare a peer relation, you should use `scenario.PeerRelation`. The core difference with regular relations is
 that peer relations do not have a "remote app" (it's this app, in fact). So unlike `Relation`, a `PeerRelation` does not
 have `remote_app_name` or `remote_app_data` arguments. Also, it talks in terms of `peers`:
 
 - `Relation.remote_units_data` maps to `PeerRelation.peers_data`
 
 ```python
-from scenario.state import PeerRelation
+import scenario
 
-relation = PeerRelation(
+relation = scenario.PeerRelation(
     endpoint="peers",
     peers_data={1: {}, 2: {}, 42: {'foo': 'bar'}},
 )
 ```
 
 be mindful when using `PeerRelation` not to include **"this unit"**'s ID in `peers_data` or `peers_ids`, as that would
-be flagged by the Consistency Checker:
+be flagged by the consistency checker:
 
 ```python
-from scenario import State, PeerRelation, Context
+import scenario
 
-state_in = State(relations=[
-    PeerRelation(
+state_in = scenario.State(relations=[
+    scenario.PeerRelation(
```

Comment:
It looks like `peer_data` is a map of unit ID to data? Again, I wonder if we can figure out a way to name the units rather than number them. Or maybe numbers are fine -- they are used in Harness testing too, for example `mysql/0`. Then again, maybe the full `mysq/0` would make it more clear it's a unit ID/name, even though the numeral is still in there.